### PR TITLE
Fix for the Jira Issue : EMM-1002

### PR DESCRIPTION
**Pull request by Kamidu**

### CODE REVIEW

File: modules/apps/jaggery/emm/src/emm/units/login/public/js/login-box.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 
-function utf8_to_b64( str ) {
-    return window.btoa(unescape(encodeURIComponent( str )));
+function utf8_to_b64(stringValue) {
```
Comment (kasundananjaya) [inline modules/apps/jaggery/emm/src/emm/units/login/public/js/login-box.js]: If these kind of methods are re-usable, can we move them to a util JS?

File: modules/apps/jaggery/emm/src/emm/units/login/public/js/login-box.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+    return window.btoa(unescape(encodeURIComponent(stringValue)));
 }
 
-function b64_to_utf8( str ) {
-    return decodeURIComponent(escape(window.atob( str )));
+function b64_to_utf8(stringValue) {
```
Comment (kasundananjaya) [inline modules/apps/jaggery/emm/src/emm/units/login/public/js/login-box.js]: If these kind of methods are re-usable, can we move them to a util JS?

File: modules/apps/jaggery/emm/src/emm/units/login/public/js/login-box.js (hunk 1 above)
Comment (Kamidu) [inline modules/apps/jaggery/emm/src/emm/units/login/public/js/login-box.js]: uill.js is a backend js file while this is a front end js file.
for increase re-usability of methods which requires from various units we can put those front js methods into a single unit and include that in every unit which requires those functionalities needed.WDYT?

File: modules/apps/jaggery/emm/src/emm/units/login/public/js/login-box.js (hunk 2 above)
Comment (Kamidu) [inline modules/apps/jaggery/emm/src/emm/units/login/public/js/login-box.js]: uill.js is a backend js file while this is a front end js file.
for increase re-usability of methods which requires from various units we can put those front js methods into a single unit and include that in every unit which requires those functionalities needed.WDYT?

**Pull request merged**